com.kristianduske.TrenchBroom.json

```diff
@@ -0,0 +1,144 @@
+{
+    "app-id": "com.kristianduske.TrenchBroom",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "trenchbroom.sh",
+    "rename-desktop-file": "trenchbroom.desktop",
+    "rename-icon": "trenchbroom",
+    "finish-args": [
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--filesystem=host",
```

Comment:
I am afraid that some Flathub maintainers may not like such high privileges.

Reply:
I know and I mentioned that in first message. The trouble is that TB didn't function as intended with more restricted privileges. For example it was unable to see some of its own files it needs to work. I'm pretty sure some solution can be cooked up but I couldn't figure it out after reading the documentation and looking json files from repository.

Reply:
If you could be more specific as to which files TB didn‘t see, we could look into this.

Reply:
I am assuming it was everything that is in `/usr/share/TrenchBroom`. I was met with empty list of configurable games, no manual, etc.